apps/web/__tests__/support/pageObjects/LoginPageObject.ts

```diff
@@ -0,0 +1,23 @@
+export class LoginPageObject {
+  forgotPassword() {
+    cy.visit('/login');
```

Comment:
I think you should use here object with link we use in app as well, so if something will change in app it will be reflected in tests as well.
```suggestion
    cy.visit(paths.authLogin);
```
and it comes from `import paths from '../../../utils/paths`, if you could replace all places where links are used, that would be great :)

Reply:
done